addons/website_blog/tests/test_performance.py

```diff
@@ -13,8 +13,8 @@ def setUp(self):
             self.env['website'].search([]).channel_id = False
 
     def test_10_perf_sql_blog_standard_data(self):
-        self.assertEqual(self._get_url_hot_query('/blog'), 25)
```

Comment:
Can't we juste assume that if the user has installed the `website` (and/or sub website module), he would like to keep this tradeoff?
Improve the website perf vs `which causes a useless LEFT JOIN on "ir_translation" in most of business flow.`

If I am not mistaken, you are prioritizing the "other business flows" when website is installed, but shouldn't we see it the other way around?
People would prefer to improve their website perf once they chose to install that module, in favor or the "other business flows"?

Reply:
Hello @rdeodoo 
I understand your concern. My goal of this PR wasn't to prioritize other flow vs website.

> If I am not mistaken, you are prioritizing the "other business flows" when website is installed, but shouldn't we see it the other way around?

This changes may be benefit for submodules of website (not always, of course) because the mixin `website.seo.metadata` is used in several models (`product.template`, `blog.tag`, `blog.post` ...) in a context where you don't need `website.seo.metadata` information.

For example, in this case/test:
We have one extra query for fetch `website.seo.metadata` of the `ir_ui_view` used (taking +- 1.5 ms in SQL), but we avoid fetching `website.seo.metadata` info of one `res.partner`, one `blog_blog`, severals `blog_tag` (save 2ms only for `blog_tag`). 
If you compare the number of LEFT JOIN to the ir_translation in this example (very naive, I know): after (148) vs before (382).


> People would prefer to improve their website perf once they chose to install that module, in favor or the "other business flows"?

Hard to answer that :smile:

Reply:
If I run this test a hundred of time, I see performance improvements (SQL time) : 0.013 vs 0.011.
(well it is just a quick test, it depends on lot of factors of course: hardware, distance of SQL server (local, the best case for the round trip), ...)

Reply:
This indeed seems to be faster, regarding the extra query, at least for `/blog` (64ms vs 68ms, average of 1000 http request on /blog).
I'd like to give it a few more tests on other flows before this got r+ again :+1: just to be sure